docs/changelogs/v0.18.md

```diff
@@ -69,16 +69,13 @@ $ curl "http://127.0.0.1:8080/ipfs/$DIR_CID?format=dag-json" | jq
 }
 ```
 
-#### Automated default `Routing.Type`
+#### Content Routing
 
-The default `Routing.Type` is changed from `dht` to `auto`.
+Content routing is the process of discovering which peers provide a piece of content. Kubo has traditionally only supported the Kademlia DHT for content routing.
 
-When `Routing.Type` is unset or set to `auto`, your node will accelerate some types of routing
-by leveraging HTTP endpoints compatible with [IPIP-337](https://github.com/ipfs/specs/pull/337)
-in addition to the IPFS DHT. DHT response timeout in `auto` mode is 5 minutes, and IPNI is 3 seconds.
-By default, an instance of IPNI ([InterPlanetary Network Indexer](https://github.com/ipni/specs/blob/main/IPNI.md#readme))
-at https://cid.contact is used.
+Kubo can now bridge networks by including support for the [delegated routing HTTP API](https://github.com/ipfs/specs/pull/337). Users can compose content routers using the `Routing.Routers` config, to pick content routers with different tradeoffs than a Kademlia DHT (for example, high-performance and high-capacity centralized endpoints, dedicated Kademlia DHT nodes, routers with unique provider records, privacy-focused content routers, etc.). One example is [InterPlanetary Network Indexers](https://github.com/ipni/specs/blob/main/IPNI.md#readme), which are HTTP endpoints that cache records from both the IPFS network and other sources such as web3.storage and Filecoin. This improves not only content availability by enabling Kubo to transparently fetch content directly from Filecoin storage providers, but also improves IPFS content routing latency by an order of magnitude and decreases resource consumption.
```

Comment:
From the suggestion before, I think we need to:
1. Caveat that this is dependent on SPs having Bitswap enabled
2. Where someone can follow along with progress with Indexers and and SP Bitswap integration.

Maybe something like:

```
*Note:* it's possible to retrieve content stored by Filecoin Storage Providers (SPs) from Kubo if the SPs service Bitswap requests.  As of this release, some SPs are advertising Bitswap.  You can follow the roadmap progress for IPNIs and Bitswap in SPs [here](https://www.starmaps.app/roadmap/github.com/protocol/bedrock/issues/1).
```

Reply:
Applied this in Incorporated in https://github.com/ipfs/kubo/pull/9475/commits/903fc25406b72850396862ba5061da075c5d1cfc